.github/dependabot.yaml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    groups:
+      docker-dependencies:
+        patterns:
+          - "*"
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    groups:
+      ci-dependencies:
+        patterns:
+          - "*"
```

Comment:
I amde sure not to add python dependency dependabot here, since it's jsut going to spam PRs, this config will just bump CI and DOckerfile deps, which are rarer